src/wiki/development/build-instructions-macos.md

```diff
@@ -0,0 +1,154 @@
+---
+eleventyNavigation:
+  key: MacOS Build instructions
+  parent: Development
+  order: 4
+---
+# Build Instructions
+
+## Contents
+
+- [Getting the source](#getting-the-source)
+- [Building](#building)
+- [IDEs and Tooling](#ides-and-tooling)
+
+## Getting the source
+
+Clone the source code using git, and grab all the submodules:
+
+```bash
+git clone --recursive https://github.com/PrismLauncher/PrismLauncher.git
+cd PrismLauncher
+```
+
+**The rest of the documentation assumes you have already cloned the repository.**
+
+## Building
+
+### Install prerequisites
+
+- Install XCode Command Line tools.
+- Install the official build of CMake (<https://cmake.org/download/>).
+- Install extra-cmake-modules
+- Install JDK 8 (<https://adoptium.net/temurin/releases/?variant=openjdk8&jvmVariant=hotspot>).
+- Install Qt 5.12 or newer or any version of Qt 6 (recommended)
+
+Using [homebrew](https://brew.sh) you can install these dependencies with a single command:
+
+```bash
+brew update # in the case your repositories weren't updated
+brew install qt openjdk@17 cmake ninja extra-cmake-modules # use qt@5 if you want to install qt5
+```
+
+### XCode Command Line tools
+
+If you don't have XCode Command Line tools installed, you can install them with this command:
+
+```bash
+xcode-select --install
+```
+
+### Build
+
+Choose an installation path.
+
+This is where the final `PrismLauncher.app` will be constructed when you run `make install`. Supply it as the `CMAKE_INSTALL_PREFIX` argument during CMake configuration. By default, it's in the dist folder, under PrismLauncher.
+
+```bash
+mkdir build
+cd build
+cmake \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_INSTALL_PREFIX:PATH="$(dirname $PWD)/dist/" \
+ -DCMAKE_PREFIX_PATH="/path/to/Qt/" \
+ -DQt5_DIR="/path/to/Qt/" \
+ -DCMAKE_OSX_DEPLOYMENT_TARGET=10.14 \
+ -DLauncher_QT_VERSION_MAJOR=6 \ # if you want to use Qt 6
+ -DENABLE_LTO=ON \ # if you want to enable LTO/IPO
+ -DLauncher_BUILD_PLATFORM=macOS
```

Comment:
Since we're at it, add -DCMAKE_BUILD_TYPE=Release/Debug here too

Reply:
In general the macos build instructions right now are pretty and outdated (e.g. they make you use make instead of ninja)

Reply:
that shouldn't be necessary with https://github.com/PrismLauncher/PrismLauncher/pull/411, though if you like I'll still add it ^^

Reply:
yeah shouldn't be necessary now